src/os/bluestore/BlueStore.cc

```diff
@@ -12709,10 +12729,27 @@ int BlueStore::queue_transactions(
   OpSequencer *osr = c->osr.get();
   dout(10) << __func__ << " ch " << c << " " << c->cid << dendl;
 
+#ifdef HAVE_JAEGER
+  jspan span = nullptr;
+  if (op && op->osd_parent_span) {
+    span = move(jaeger_tracing::child_span("BlueStore::queue_transactions",
```

Comment:
I am surprised that move() does something useful on unnamed temporary object.
It improves performance, or it is here just to signal reader that move assignment operator will be invoked?

Reply:
i don't think we should use `move()` here. clang even warns at seeing this if `-Wpessimizing-move` is enabled.

Reply:
thank you @tchaikov @aclamk. `span = move(jaeger_tracing::child_span(` is removed from the latest code.